packages/monaco/README.md

```diff
@@ -19,6 +19,62 @@ This includes:
 - code snippets
 - textmate grammars (theme registry, service)
 
+## Monaco Uplifts
+
+This package is intended to be the interface between the `@theia/monaco-editor-core` package, the project's bundling of the `monaco-editor-core` package published by the VSCode
+team, and the rest of the application. When we uplift to a new version of `monaco-editor-core`, this package will need to be checked particularly thoroughly. To facilitate that
+process, the steps for undertaking a Monaco uplift are outlined here.
+
+### Setting up the VSCode side
+
+1. Clone the VSCode repo and make sure you have the following remotes:
+ - https://github.com/microsoft/vscode.git - the official VSCode repo.
+ - https://github.com/theia-ide/vscode.git - Theia's fork.
+2. Find the latest release tag in the official VSCode repo, and the most recent uplift branch in the Theia fork.
+ > At the time of this writing the latest release tag is `1.67.2`, and the uplift branch is `monaco-uplift-2022-6`
+3. Check out the release tag, cherry pick the tip of the uplift branch, and resolve any conflicts.
```

Comment:
Usually, one cherry-picks commits, not branches. Which commits do you mean, concretely?

Reply:
In the git CLI, `git cherry-pick <branch name>` is equivalent to cherry-picking the tip of that branch, which is the intended effect of this step. See [this example](https://git-scm.com/docs/git-cherry-pick#Documentation/git-cherry-pick.txt-codegitcherry-pickmastercode).